packages/unity-interface/BrowserInterface.ts

```diff
@@ -379,13 +416,34 @@ export class BrowserInterface {
         }
       }
       store.dispatch(saveProfileDelta(update))
+    } else if (validateRendererSaveProfileV0(changes as RendererSaveProfile)) {
+      const update: Partial<Avatar> = {
+        avatar: {
+          bodyShape: changes.avatar.bodyShape,
+          eyes: { color: changes.avatar.eyeColor },
+          hair: { color: changes.avatar.hairColor },
+          skin: { color: changes.avatar.skinColor },
+          wearables: changes.avatar.wearables,
+          emotes: (changes.avatar.emotes ?? []).map((value, index) => ({ slot: index, urn: value as any as string })),
+          snapshots: {
+            body: changes.body,
+            face256: changes.face256
+          }
+        }
+      }
+      store.dispatch(saveProfileDelta(update))
     } else {
-      trackEvent('invalid_schema', { schema: 'SaveUserAvatar', payload: changes })
+      defaultLogger.error('error validating schema', validateRendererSaveProfileV1.errors)
+      trackEvent('invalid_schema', {
+        schema: 'SaveUserAvatar',
+        payload: changes,
+        errors: (validateRendererSaveProfileV1.errors ?? []).map(($) => $.message).join(',')
```

Comment:
Should we add logic here to know which `validate` fn failed? 
The `else` here will only  show the `validateRendererSaveProfileV1` errors even if `validateRendererSaveProfileV0` was the that failed.